[2024-03-02T18:56:32.397+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-02T18:56:32.412+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-03-02T18:56:32.413+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-03-02T18:56:32.553+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_html_data> on 2024-01-02 00:00:00+00:00
[2024-03-02T18:56:32.559+0000] {standard_task_runner.py:60} INFO - Started process 308 to run task
[2024-03-02T18:56:32.562+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_ncei_data', 'download_html_data', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/sample.py', '--cfg-path', '/tmp/tmpcuamzfw0']
[2024-03-02T18:56:32.564+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask download_html_data
[2024-03-02T18:56:32.749+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-02T00:00:00+00:00 [running]> on host a890577b40a7
[2024-03-02T18:56:32.970+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='fetch_ncei_data' AIRFLOW_CTX_TASK_ID='download_html_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-03-02T18:56:32.972+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-02T18:56:32.975+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html"]
[2024-03-02T18:56:32.987+0000] {subprocess.py:86} INFO - Output:
[2024-03-02T18:56:32.989+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: syntax error near unexpected token `('
[2024-03-02T18:56:32.990+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: `curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html'
[2024-03-02T18:56:32.991+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-03-02T18:56:33.007+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-03-02T18:56:33.013+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=fetch_ncei_data, task_id=download_html_data, execution_date=20240102T000000, start_date=20240302T185632, end_date=20240302T185633
[2024-03-02T18:56:33.737+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 29 for task download_html_data (Bash command failed. The command returned a non-zero exit code 2.; 308)
[2024-03-02T18:56:33.781+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-02T18:56:33.947+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
